libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-participant-certificate.store.ts

```diff
@@ -41,17 +41,18 @@ const initialState: CertificateState = {
 export class DhMarketParticipantCertificateStore extends ComponentStore<CertificateState> {
   private readonly httpClient = inject(MarketParticipantActorHttp);
```

Comment:
Question: What would you set on httpClient?

Reply:
Is the question about the `readonly` keyword? 

Hopefully nothing that can break the service. I guess it makes sense to use `readonly` when building a library that will be consumed by other users. Not sure if it is as useful when we know that the service will only be used by us.

Reply:
Yes it was :-) You can keep it as is, were just curious if I were missing something.